portal/src/graphql/portal/LoginMethodConfigurationScreen.tsx

```diff
@@ -2261,6 +2317,24 @@ function VerificationSettings(props: VerificationSettingsProps) {
             selectedKey={emailRatelimitConfig.resend_cooldown_seconds}
             onChange={onChangeEmailResendCooldown}
           />
+          <Toggle
+            inlineLabel={true}
+            checked={
+              authenticatorOOBEmailConfig.secondary_allow_unverified ?? false
+            }
+            onChange={onChangeEmailSecondaryAllowUnverified}
+            label={renderToString(
+              "VerificationConfigurationScreen.verification.email.secondary-allow-univerified.label"
+            )}
+          />
+          <Dropdown
+            label={renderToString(
+              "VerificationConfigurationScreen.verification.email.secondary-verify-by.label"
+            )}
+            options={emailOTPModes}
+            selectedKey={authenticatorOOBEmailConfig.secondary_email_otp_mode}
+            onChange={onChangeSecondaryEmailOTPMode}
+          />
```

Comment:
Magic link is for the secondary authenticator only, maybe we should set this in the 2FA page instead of login method?

Reply:
Updated